friendships/services.py

```diff
@@ -31,4 +31,11 @@ def get_followers(cls, user):
         friendships = Friendship.objects.filter(
             to_user=user,
         ).prefetch_related('from_user')
-        return [friendship.from_user for friendship in friendships]
\ No newline at end of file
+        return [friendship.from_user for friendship in friendships]
+
+    @classmethod
+    def has_followed(cls, from_user, to_user):
```

Comment:
to show the whether current user has followed the friend of someone's friend list